ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource

```diff
@@ -138,6 +138,15 @@ Verify There Is No "Error Displaying Pipelines" After Creating Pipeline Server
     Run Keyword And Ignore Error       Wait Until Page Contains    Error displaying pipelines    timeout=60s
     Run Keyword And Warn On Failure    Page Should Not Contain     Error displaying pipelines
 
+Verify That There Are No Sample Pipelines After Creating Pipeline Server
```

Comment:
I believe in ODH the pipeline samples will be available after creating a pipeline server. You could 
call your keyword `Verify Sample Pipelines Existence After Creating Pipeline Server` and fail (not log) if the samples are created for RHOAI .

Something like this (not tested):
```
${samples_created}=      Run Keyword And Return Status   Wait Until Page Contains    Create pipeline    timeout=10s
IF  "${PRODUCT}" == "RHODS" and ${samples_created}
    Fail    Default Sample Pipelines should not be created after creating a pipeline server in ${PRODUCT}
ELSE IF     "${PRODUCT}" == "ODH" and not ${samples_created}
    Fail    Default Sample Pipelines should be created after creating a pipeline server in ${PRODUCT}
END
```

Reply:
> I believe in ODH the pipeline samples will be available after creating a pipeline server. You could

if so, then that's a bug, https://issues.redhat.com/browse/RHOAIENG-4382, @HumairAK is working on removing the samples from both ODH and RHOAI

Reply:
Yes, @jiridanek  modified the test to fail if samples are available in either RHOAI or ODH